### PR TITLE
feat(core): Expand Section Breaks to better search doctype

### PR DESCRIPTION
**Pull request by DeeMysterio**
collapse-sections

### CODE REVIEW

**Review by Er-Naren719 — APPROVED** [review]
Code looks good.

**Comment by vjFaLk** [comment]
@DeeMysterio Please also make this to Frappe Upstream. Take @Alchez's help.

File: frappe/public/js/frappe/form/toolbar.js [hunk 1]
```diff
@@ -182,13 +182,27 @@ frappe.ui.form.Toolbar = Class.extend({
...
+				
+				// Expand all sections
+				this.page.add_menu_item(__("Expand All Sections"), function() {
+					var all_fields_in_doc = me.frm.meta.fields;
+					function returnCollapsibleFields(field) { 
+						if (field.collapsible == 1 ) {
+							return field
+						}
+					}
+					var collapsible_fields = all_fields_in_doc.filter(returnCollapsibleFields);
```
Comment (Alchez) [inline frappe/public/js/frappe/form/toolbar.js]: Get rid of the function, do it inline.

```suggestion
					let collapsible_fields = all_fields_in_doc.filter((field) => field.fieldtype == "Section Break");
```

**Review by Alchez — CHANGES_REQUESTED** [review]

File: frappe/public/js/frappe/form/toolbar.js [hunk 2]
```diff
@@ -182,13 +182,27 @@ frappe.ui.form.Toolbar = Class.extend({
 			}
 		}
 
-		// feedback
+		// Request feedback
 		if(!this.frm.doc.__unsaved) {
 			if(is_submittable && docstatus == 1) {
```
Comment (Alchez) [inline frappe/public/js/frappe/form/toolbar.js:187]: Allow expanding the sections even for Draft (and maybe unsaved) documents.

**Review by Alchez — CHANGES_REQUESTED** [review]

**Pull request merged**